docs/mdbook/default.nix

```
@@ -54,11 +53,11 @@ let
         if isOpt then
           opts
         else
-          filterAttrs (_: component: component.isOption && (isVisible component)) opts;
-      path = removeWhitespace (concatStringsSep "/" path);
+          lib.filterAttrs (_: component: component.isOption && (isVisible component)) opts;
+      path = removeWhitespace (lib.concatStringsSep "/" path);
       moduleDoc =
         let
-          info = optionalAttrs (hasAttrByPath path nixvimInfo) (getAttrFromPath path nixvimInfo);
+          info = lib.optionalAttrs (lib.hasAttrByPath path nixvimInfo) (lib.getAttrFromPath path nixvimInfo);
```

If we're touching this line anyway, may as well improve it to use [`attrByPath`](https://noogle.dev/f/lib/attrByPath):
```suggestion
          info = lib.attrByPath path { } nixvimInfo;
```